src/library/scala/collection/IterableOnce.scala

```diff
@@ -43,37 +43,37 @@ final class IterableOnceExtensionMethods[A](private val it: IterableOnce[A]) ext
     case _ => it.iterator.foreach(f)
   }
 
-  @deprecated("Use factory.from(it) instead of it.to(factory) for IterableOnce", "2.13.0")
+  @deprecated("Use it.iterator.to(factory) instead of factory.from(it) for IterableOnce", "2.13.0")
```

Comment:
I would rather omit the `it` identifier and the end of the sentence: “Use `.iterator.to(factory)”.

What do you think?

Reply:
Yes, I agree -- this will remove some clutter from the recommendation